src/Idle.DataAccess/Repositories/RepositoryBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Idle.DataAccess.Common;
+using Idle.DataAccess.Fields;
+using Idle.DataAccess.Fields.Languages;
+using SQLite;
+using System.Text;
+
+namespace Idle.DataAccess.Repositories
+{
+
+
+	public abstract class RepositoryBase
+    {
+        protected internal SQLiteOpenFlags Flags;
+        protected static SQLiteAsyncConnection _connection;
+
+        protected RepositoryBase()
+        {
+            Ctor();
+            _connection = new SQLiteAsyncConnection(Constants.DatabasePath, Constants.Flags);
```

Comment:
We are not passing in the SQLiteOpenFlags `Flags` here, its taking in the `Constants.Flags` boolean as `SQLiteAsyncConnection(string databasePath, bool storeDateTimeAsTicks)`, rather than `SQLiteAsyncConnection(string databasePath, SQLiteOpenFlags openFlags)`.